GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -166,6 +166,9 @@ void Mesh::generateVAO() {
         return;
     }
 
+    GLuint vaoID_ = 0;
+	GLuint triangle_vboID_, vert_vboID_, norm_vboID_, tex_vboID_;
```

Comment:
I do not see any indenting on my PC. And it seems Shift+Ctrl+F does not work correctly for CPP files. Maybe my formatter setting has any problems?